static/app/components/feedback/list/issueTrackingSignals.tsx

```diff
@@ -27,20 +33,27 @@ function filterLinkedPlugins(actions: ExternalIssueComponent[]) {
   return plugins.concat(nonPlugins);
 }
 
-function getPluginNames(pluginIssue) {
+function getPluginNames(pluginIssue: PluginIssueComponent | PluginActionComponent) {
   return {
     name: pluginIssue.props.plugin.name ?? '',
     icon: pluginIssue.props.plugin.slug ?? '',
   };
 }
 
-function getIntegrationNames(integrationIssue) {
+function getIntegrationNames(integrationIssue: IntegrationComponent) {
   return {
     name: integrationIssue.props.configurations[0].provider.name ?? '',
     icon: integrationIssue.key ?? '',
   };
 }
 
+function getAppIntegrationNames(integrationIssue: SentryAppIssueComponent) {
+  return {
+    name: integrationIssue.props.sentryApp.name,
```

Comment:
i'm not sure if this is the right/best name to show. But i think before we had no name for the `SentryAppIssueComponent` case.